ocaml/idl/datamodel_vtpm.ml

```diff
@@ -42,7 +55,7 @@ let t =
       ; (Extended, rel_next, "Added VTPM profiles")
       ; (Changed, rel_next, "Removed backend field")
       ]
-    ~gen_constructor_destructor:true ~name:_vtpm ~descr:"A virtual TPM device"
+    ~gen_constructor_destructor:false ~name:_vtpm ~descr:"A virtual TPM device"
```

Comment:
I assume you need to disable autogeneration because you want to override them to add the assertions?

Reply:
indeed, otherwise it's warnings galore in the server

Reply:
Hmmm... what's this got to do with anything in the implementation of the module? What sort of warnings in server.ml?

Reply:
it creates identical branches when matching for the api call used, so ocaml warns about one of them never being used

Reply:
If you still add the separate function to the `messages` below, yes. But why do you need that?

Reply:
All this is to expose them to the API, couldn't find a way to explicitely expose them without defining them

Reply:
The local index.html generated doesn't show implicit messages, with was confusing because there were available in the API regardless. I've removed the commit since it wasn't useful.